### PR TITLE
Added Build Number in settings

### LINKED ISSUE #3736: Add the build number in addition to version number in settings
Add the build number in addition to version number in settings. That will be useful to debug the internal builds in some cases.

### PR DESCRIPTION
**Pull request by mrudultora**
####  Description
Added _versionCode / Build Number_ in Settings ( Preferences Fragment ).

#### Related issues
Fixes #3736 

#### Related PRs
None

#### Screenshots

### CODE REVIEW

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/features/PreferencesFragment.kt [hunk 1]
```diff
@@ -240,7 +240,12 @@ class PreferencesFragment : PreferenceFragmentCompat(), INavigationItem, OnShare
             try {
                 val pInfo = requireActivity().packageManager.getPackageInfo(requireActivity().packageName, 0)
                 val version = pInfo.versionName
-                it.summary = getString(R.string.version_string) + " " + version
+                val versionCode = if(Build.VERSION.SDK_INT>=Build.VERSION_CODES.P){
+                    pInfo.longVersionCode
+                } else {
+                    pInfo.versionCode
+                }
+                it.summary = getString(R.string.version_string) + " " + version+" ("+ versionCode + ")"
```
Comment (VaiTon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/features/PreferencesFragment.kt]: Please use string template.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/features/PreferencesFragment.kt [hunk 2]
```diff
@@ -240,7 +240,12 @@ class PreferencesFragment : PreferenceFragmentCompat(), INavigationItem, OnShare
             try {
                 val pInfo = requireActivity().packageManager.getPackageInfo(requireActivity().packageName, 0)
                 val version = pInfo.versionName
-                it.summary = getString(R.string.version_string) + " " + version
+                val versionCode = if(Build.VERSION.SDK_INT>=Build.VERSION_CODES.P){
```
Comment (VaiTon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/features/PreferencesFragment.kt]: Please format the file

**Review by VaiTon — CHANGES_REQUESTED** [review]

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/features/PreferencesFragment.kt (hunk 1 above)
Comment (mrudultora) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/features/PreferencesFragment.kt:248]: How will I use string template here ?

**Comment by mrudultora** [comment]
@VaiTon  Are the changes correct ?

**Review by VaiTon — APPROVED** [review]

**Comment by VaiTon** [comment]
> 
> 
> @VaiTon Are the changes correct ?

See latest commit

**Pull request merged**